### DISCUSSION TITLE
Flink SQL Connector for SQL Server CDC version 2.4.0, connecting to SQL Server 2012 with authorized user, but encountering continuous errors when accessing CDC

### DISCUSSION THREAD

**Discussion by BearBabyLiu** [apache/flink-cdc#2946] [Q&A]
with the Flink SQL connector for SQL Server CDC version 2.4.0. The problem mentioned is related to connecting to a SQL Server 2012 instance where the user is already authorized, but there are continuous errors when trying to access the Change Data Capture (CDC) feature.
![image](https://github.com/ververica/flink-cdc-connectors/assets/20611608/3d1f0291-5e95-45e0-86e4-3036f6a04f62)
![image](https://github.com/ververica/flink-cdc-connectors/assets/20611608/13f769ce-35c3-4f66-a3cf-589f570c0d90)
"May I ask where the issue lies? Has anyone else encountered this?very urgent，thank you！

**Comment by BearBabyLiu** [comment]
@leonardBang 麻烦了，还有flinkcdc的社群满，还可以邀请我进去吗？谢谢了

**Reply by BearBabyLiu** [reply]
@GOODBOY008  Hello, you are very professional. Could you please take a look?

**Reply by GOODBOY008** [reply]
Hi @BearBabyLiu ,You use `seatunnel` ?
